jsapp/js/account/DeleteAccountBanner.stories.tsx

```diff
@@ -17,7 +16,11 @@ const meta: Meta<typeof DeleteAccountBanner> = {
   argTypes: {},
   parameters: {
     msw: {
-      handlers: [assetsMock, meMock, environmentMock, organizationMock(meMockResponse.organization!.uid)],
+      handlers: {
+        me: meMockResponse,
```

Comment:
@Akuukis why are we adding `meMockResponse`? It's just a `const`, right? Plus we already have `meMock` being added in `preview.tsx` to all the stories

Reply:
right, that's redundant :ok_hand: